js-apps/meep-frontend/src/js/components/dialogs/id-dialog.js

```diff
@@ -28,7 +28,7 @@ import {
 const IDDialog = props => {
   return (
     <Dialog
-      style={{zIndex: 1000}}
+      style={{zIndex: 10000}}
       open={props.open}
```

Comment:
Not sure how zIndex 10000 relates to this bugfix.
Double-check if it's intended.

Reply:
This is a minor frontend fix that allows the dialogs to appear above the map control buttons. It was intended but not mentioned in the PR.